async_approach.py

```diff
@@ -0,0 +1,43 @@
+import argparse
+import asyncio
+import aiohttp
+import requests
+import time
+import re
+
+URL = 'https://wallpaperscraft.ru/'
+
+
+def get_image_urls(url, image_format='jpg', image_amount=5):
+    return list(re.findall(r'https:[\S]*\.' + f'{image_format}', requests.get(url).text))[:image_amount]
```

Comment:
Я так понимаю, вы скачиваете все изображения с какого то сайта. Неплохо

Reply:
Считает, что так быстрее
```suggestion
def get_image_urls(url, image_format='jpg', image_amount=5):
    return [*re.findall(r'https:[\S]*\.' + f'{image_format}', requests.get(url).text)][:image_amount]
```